src/test/data/README.md

```diff
@@ -0,0 +1,124 @@
+# Input Data Format Requirements
+
+This project is currently supporting input of *Rank 1 Constraint Systems* (soon to support input of Trusted Setup and Proving Key) for plain-text and JSON file types (eventually binary).
```

Comment:
I really like the elaborate README! Maybe you could do another pass to make sure it is up to date.

Maybe we should not say what will be soon supported.